fs-storage/Cargo.toml

```diff
@@ -9,3 +9,17 @@ crate-type = ["rlib"]
 bench = false
 
 [dependencies]
+data-error = { path = "../data-error" }
+data-json = { path = "../data-json" }
+fs-atomic-versions = { path = "../fs-atomic-versions" }
+data-resource = { path = "../data-resource" }
+log = { version = "0.4.17", features = ["release_max_level_off"] }
+
+serde_json = "1.0.82"
+serde = { version = "1.0.138", features = ["derive"] }
+
+[dev-dependencies]
+log = { version = "0.4.17", features = ["release_max_level_off"] }
+tempdir = "0.3.7"
+tempfile = "3.10.1"
```

Comment:
We use `tempdir` in tests for other crates. 
If there is no reason to prefer `tempfile`, we shouldn't use it only in this crate